ark/mainnet/knownWallets.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Why would this be here? This is explorer specific and not used for anything else.